blog/2022/12-7-first-look-at-resoto-dashboards/index.md

```diff
@@ -0,0 +1,83 @@
+---
+authors: [pablo]
+tags: [ui, dashboard, visualization]
+---
+
+# First look at resoto dashboards
+
+Resoto makes it easier than ever to create an inventory of your cloud infrastructure. And with just a few commands, you can immediately get a snapshot of what is currently happening with your resources. Furthermore, with this data ingested into a time series database, you not only know the current state of your cloud but also how they evolve.
```

Comment:
Maybe add a short introductory paragraph that specifies the problem?

Getting a general overview of all your cloud resources is useful. But sometimes you want to configure a custom view, for the type of resources that matter for your use case. We're excited to introduce Resoto Dashboards - responsive and configurable dashboards.  In this article, I'm showing you how you can use Resoto Dashboards to:

1. configure data sources and widgets that display the data
2. etx.
3. etc.

- 



Resoto makes it easy to create an inventory of your cloud infrastructure, aka "[cloud asset inventory](https://resoto.com/blog/2022/07/28/what-is-cloud-asset-inventory)". With a few